### ISSUE TITLE
Step missing before starting the containers

### ISSUE THREAD

**Issue by felix-d** [CrowdHailer/water-cooler.elixir#16]
Hi! Thanks for this repo and your great presentation!

`mix.deps` is not part of the Dockerfile so when running `docker-compose up` the first time I got this exception

```
☁  elixir-on-docker  docker-compose up
Starting elixirondocker_www_1 ...
Starting elixirondocker_www_1 ... done
Attaching to elixirondocker_www_1
www_1  | Unchecked dependencies for environment dev:
www_1  | * exsync (Hex package)
www_1  |   the dependency is not available, run "mix deps.get"
www_1  | * server_sent_event (Hex package)
www_1  |   the dependency is not available, run "mix deps.get"
www_1  | * wobserver (Hex package)
www_1  |   the dependency is not available, run "mix deps.get"
www_1  | * raxx_static (Hex package)
www_1  |   the dependency is not available, run "mix deps.get"
www_1  | * ace (Hex package)
www_1  |   the dependency is not available, run "mix deps.get"
www_1  | * raxx_api_blueprint (Hex package)
www_1  |   the dependency is not available, run "mix deps.get"
www_1  | ** (Mix) Can't continue due to errors on dependencies
elixirondocker_www_1 exited with code 1
```

I had to run `docker-compose run --rm www mix deps.get` first then it worked. This is not mentioned in the Readme.

**Comment by felix-d** [comment]
Nevermind I saw you added that further down the Readme. My bad.

**Comment by CrowdHailer** [comment]
Hey if you think it can be clearer in the readme feel free to tweek it

**Comment by felix-d** [comment]
There you go :) https://github.com/CrowdHailer/elixir-on-docker/pull/18

I was confused at first because the same container is used for both development and deployment in your talk, so you need to make sure to run `mix deps.get` before rebuilding the image and pushing to the docker hub.  If the Dockerfile was only used for deployment, the `RUN mix deps.get` would probably be in the Dockerfile itself.

**Comment by felix-d** [comment]
Oops hit the button ^

**Comment by ream88** [comment]
I encountered the same problem! @CrowdHailer could you please explain why you need to run `mix deps.get` although it's already included in the Dockerfile?

**Comment by GPrimola** [comment]
I'm facing the same issue here..
It seems that the folder `deps/` created by `mix deps.get` isn't being kept by the build of the container.

Does anyone has any clue for that?

**Comment by CrowdHailer** [comment]
@ream88 If i remember correctly it is so the deps are saved to the host machine so that they can be copied from the host rather than pulled from hex in subsequent builds
